Packs/ServiceNow/ReleaseNotes/2_1_4.md

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
please add an entry for the script update of the DO

Reply:
also mention you removed the display and escalation args as they did not ever work

Reply:
done